src/phoneField/PhoneField.story.mdx

```diff
@@ -0,0 +1,96 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks'
+
+import { PhoneField } from './index'
+
+<Meta title="Forms|PhoneField" />
+
+# **PhoneField**
+
+<Preview>
+  <Story name="Default">
+    <label id="phoneField" htmlFor="phoneField">
+      Accessibility label
+    </label>
+    <PhoneField id="phoneField" name="phoneField" onChange={() => {}} />
+  </Story>
+</Preview>
+
+### Custom
+
+<Preview>
+  <Story name="Custom">
+    <React.Fragment>
+      <label id="phoneFieldId" htmlFor="phoneFieldId">
```

Comment:
Remove the `htmlFor="phoneFieldId"` on the label tag. 
Remove `id="phoneFieldId"` as already on the `label` (need to be unique).
Update `aria-labelledby="labelId"` to `aria-labelledby="phoneFieldId"` on the `PhoneField ` component.

Reply:
To fix the linter error you could replace the <label> with a <heading> and it'll more correspond to the way we use the `PhoneField` component.